packages/manager/modules/email-domain/src/mailing-list/subscribers/create/email-domain-mailing-list-subscribers-create.controller.js

```diff
@@ -29,11 +29,15 @@ export default class MailingListsCreateSubscriberCtrl {
     this.loading = true;
     const subscribersToAdd = uniq(this.selection);
 
-    return this.MailingLists.addSubscribers(this.$stateParams.productId, {
-      mailingList: this.mailingList.name,
-      users: subscribersToAdd,
-      type: 'subscriber',
-    })
+    return this.MailingLists.addSubscribers(
+      this.$stateParams.productId,
+      {
+        mailingList: this.mailingList.name,
+        users: subscribersToAdd,
+        type: 'subscriber',
+      },
+      75,
```

Comment:
**nitpick:** Can be maintained in a constant file.

Reply:
As seen in private, because i used it only in one place, i don't create a variable for that (no reusable).